src/Core/src/Graphics/PaintExtensions.Android.cs

```diff
@@ -84,7 +84,7 @@ internal static bool IsSolid(this AColor color)
 
 		internal static bool IsSolid(this SolidPaint paint)
 		{
-			return paint.Color.Alpha == 1;
+			return paint?.Color != null && paint.Color.Alpha == 1;
```

Comment:
Could include some unit tests https://github.com/dotnet/maui/blob/f126268ef630f9527e0f16d65c12acc39ff933a2/src/Core/tests/DeviceTests/Graphics/GraphicsTests.Android.cs#L154 checking for null values?